packages/esm-commons-lib/src/components/banner-tags/active-visit-tag.component.tsx

```diff
@@ -1,30 +0,0 @@
-import React from 'react';
-import { Tag, Tooltip } from '@carbon/react';
-import { useVisit } from '@openmrs/esm-framework';
-import dayjs from 'dayjs';
-import { useTranslation } from 'react-i18next';
-import styles from './active-visit-tag.scss';
```

Comment:
Thanks @kirwea for this, kindly delete the styles file as well, i don't think its being used